### PR TITLE
add recipe for liip/monitor-bundle:2.6.*

### PR DESCRIPTION
**Pull request by lsmith77**
| Q             | A
| ------------- | ---
| License       | MIT

my very first flex recipe ..

### CODE REVIEW

**Comment by lsmith77** [comment]
FYI @kbond

File: liip/monitor-bundle/2.6/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+  <bg=blue;fg=white>         Getting started using </><bg=blue;fg=white;options=bold>liip/monitor-bundle</><bg=blue;fg=white>          </>
+
+  <fg=blue;options=bold>Configure</> <fg=blue>your transformations:</>
```
Comment (Nyholm) [inline liip/monitor-bundle/2.6/post-install.txt]: We should try to keep colors to a minimum.

**Comment by Nyholm** [comment]
Great! Just make sure Travis is happy and I'll be happy to merge.

**Comment by lsmith77** [comment]
lets see if merging https://github.com/liip/LiipMonitorBundle/pull/165 and tagging a new release will do the trick :)

**Comment by lsmith77** [comment]
@Nyholm green :)

File: liip/monitor-bundle/2.6/config/packages/monitor.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+liip_monitor:
+    # enabling the controller requires that `assets` are enabled in the framework bundle
```
Comment (fabpot) [inline liip/monitor-bundle/2.6/config/packages/monitor.yaml]: Would it make sensio to auto-enable the controller based on the presence of assets (in a compiler pass in the bundle)?

Comment (lsmith77) [reply liip/monitor-bundle/2.6/config/packages/monitor.yaml]: no .. I think most people will actually never enable the controller and only use this Bundle via the CLI

**Pull request merged**